README.md

```diff
@@ -78,26 +78,27 @@ These steps only work for operators registered on-chain!
 
 3. Run the service:
 
-      - Option 1: Step-by-step
+      - **Option 1: Step-by-step**
 
       Ensure you have set the following environment variables:
 
       ```bash
-      export SKILL_ELCOLLECTOOORR_ABCI_MODELS_PARAMS_ARGS_SETUP_SAFE_CONTRACT_ADDRESS=["0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a"]
-      export SKILL_ELCOLLECTOOORR_ABCI_MODELS_PARAMS_ARGS_WHITELISTED_INVESTOR_ADDRESSES={YOUR_WHITELIST}
-      export SERVICE_ELCOLLECTOOORR_RPC={YOUR_RPC_URL}
+      export SKILL_ELCOLLECTOOORR_ABCI_MODELS_PARAMS_ARGS_SETUP_SAFE_CONTRACT_ADDRESS=`["0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a"]`
+      export SKILL_ELCOLLECTOOORR_ABCI_MODELS_PARAMS_ARGS_WHITELISTED_INVESTOR_ADDRESSES='["YOUR_WHITELIST"]'
+      export SERVICE_ELCOLLECTOOORR_RPC_0="YOUR_RPC_URL"
+      export SERVICE_ELCOLLECTOOORR_RPC_1="YOUR_RPC_URL"
       ```
 
-      where `0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a` should match the correct address from the on-chain service deployment, and `{YOUR_WHITELIST}` and `{YOUR_RPC_URL}` should be replaced accordingly.
+      where `0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a` should match the correct address from the on-chain service deployment, and `YOUR_WHITELIST`, `YOUR_RPC_URL_0` and `YOUR_RPC_URL_1` should be replaced accordingly.
 
-      Then fetch the service
+      Then fetch the service:
 
       ```bash
-      autonomy fetch elcollectooorr/elcollectooorr:0.1.0:bafybeifhe4apmnehbfke7dgtghxbv6775nix56b2x66uhilixssuvjre5u --service
+      autonomy fetch elcollectooorr/elcollectooorr:0.1.0:bafybeiaoz56cvp5bl3xpsgyqomtf4umfrsm7qab7andq4gar6qwf6jcqzi --service
       cd elcollectooorr
       ```
 
-      Replace the API key placeholder for the RPC endpoint.
+      Then build the service:
 
       ```bash
       autonomy build-image
```

Comment:
It builds fine for me, but not with the right aea version:
<img width="538" alt="Screenshot 2022-11-05 at 10 37 41" src="https://user-images.githubusercontent.com/19314578/200116199-7d3d052d-3eec-4676-9706-7683ef1252c6.png">

Reply:
How can we get it to force-use the latest?

Reply:
Having it always use latest might lead to breaking builds. Consider the following scenario:
- User has open-autonomy@v0.3.5 installed, and their packages are built for this version.
- We update open-autonomy to v0.4.0, and we have introduced braking changes. 
Because the user has open-autonomy@v0.3.5, and it always uses the latest image, when the user would build their image, they would be building with the latest image (v0.4.0) their packages which are not compatible with this version.
I think we should start tagging the images to reflect the version of autonomy they are built for, in addition to tagging new releases with `latest`.

Reply:
Agree - that's actually what I ended up implementing here: https://github.com/valory-xyz/open-autonomy/pull/1520